config/features.yml

```diff
@@ -407,6 +407,14 @@ features:
     actor_type: user
     description: Ask for a password when an encrypted PDF is detected before uploading
     enable_in_development: true
+  rrd_hypertension_compensation:
+    actor_type: user
+    description: Fast tracks 526 disability compensation hypertension claims by submitting additional health data
+    enable_in_development: true
+  rrd_add_special_issue:
+    actor_type: user
+    description: Fast tracks 526 disability compensation claims by adding the RRD special issue to the claim.
+    enable_in_development: true
```

Comment:
Copied from `disability_hypertension_compensation_fast_track` and `disability_hypertension_compensation_fast_track_add_rrd`, respectively.